sp800-63b/references.md

```diff
@@ -63,52 +63,52 @@
 
```

Comment:
I just picked one of the formats you had used previously. In researching, I had seen it listed several different ways. Same goes with repeating the RFC number. I am indifferent, was just trying to be consistent.